content/en/news/security/istio-security-2022-003/index.md

```diff
@@ -31,30 +31,42 @@ For simple installations, istiod is typically only reachable from within the clu
+  (CVSS Score 6.5, Medium): Potential null pointer dereference when using JWT filter `safe_regex` match.
 
-- __[CVE-2021-43825](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2021-43825])__:
-  CVE-2021-43825 (CVSS Score 6.1, Medium): Envoy 1.21.0 and earlier - Use-after-free when response filters increase response data, and increased data exceeds downstream buffer limits.
+- __[CVE-2021-43825](https://github.com/envoyproxy/envoy/security/advisories/GHSA-h69p-g6xg-mhhh])__:
+  (CVSS Score 6.1, Medium):  Use-after-free when response filters increase response data, and increased data exceeds downstream buffer limits.
 
-- __[CVE-2021-43826](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2021-43826])__:
-  CVE-2021-43826 (CVSS Score 6.1, Medium): Envoy 1.21.0 and earlier - Use-after-free when tunneling TCP over HTTP, if downstream disconnects during upstream connection establishment.
+- __[CVE-2021-43826](https://github.com/envoyproxy/envoy/security/advisories/GHSA-cmx3-fvgf-83mf])__:
+  (CVSS Score 6.1, Medium): Use-after-free when tunneling TCP over HTTP, if downstream disconnects during upstream connection establishment.
 
-- __[CVE-2022-21654](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2022-21654])__:
-  CVE-2022-21654 (CVSS Score 7.3, High): Envoy 1.7.0 and later - Incorrect configuration handling allows mTLS session re-use without re-validation after validation settings have changed.
+- __[CVE-2022-21654](https://github.com/envoyproxy/envoy/security/advisories/GHSA-5j4x-g36v-m283])__:
+  (CVSS Score 7.3, High): Incorrect configuration handling allows mTLS session re-use without re-validation after validation settings have changed.
 
-- __[CVE-2022-21655](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2022-21655])__:
-  CVE-2022-21655 (CVSS Score 7.5, High): Envoy 1.21 and earlier - Incorrect handling of internal redirects to routes with a direct response entry.
+- __[CVE-2022-21655](https://github.com/envoyproxy/envoy/security/advisories/GHSA-7r5p-7fmh-jxpg])__:
+  (CVSS Score 7.5, High): Incorrect handling of internal redirects to routes with a direct response entry.
 
-The following CVE did not apply to Istio 1.11.6.
+The following CVEs only affected Istio 1.12.0-1.12.3 and 1.13.0. It did not affect Istio 1.11.
 
-- __[CVE-2022-23606](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=CVE-2022-23606])__:
-  CVE-2022-23606 (CVSS Score 4.4, Moderate): Envoy 1.20 and later - Stack exhaustion when a cluster is deleted via Cluster Discovery Service.
+- __[CVE-2022-23606](https://github.com/envoyproxy/envoy/security/advisories/GHSA-9vp2-4cp7-vvxf])__:
+  (CVSS Score 4.4, Moderate): Stack exhaustion when a cluster is deleted via Cluster Discovery Service.
+
+The following CVEs did not have patches for the version of Envoy used in Istio 1.11. While the CVE exists, Istio
+does not believe that it is affected by this issue.
```

Comment:
Do the CVEs affect 1.12 and 1.13? A Table may be good for which fixes exist and what versions are effected.

Reply:
Added a table